ccm_web/client/src/components/SectionSelectorWidget.tsx

```diff
@@ -594,7 +599,7 @@ function SectionSelectorWidget (props: ISectionSelectorWidgetProps): JSX.Element
                 divider
                 disableGutters
                 classes={{ root: classes.listItemRoot }}
-                selected={isSelected}
```

Comment:
Did this changed fixed the message saying "Unmerging"?

Reply:
I think Unmerging issue #408 seems to solved. is this the change that fixed it?

Reply:
Yes, this was to to address the unmerging bug. I think secondaryAction was introduced for ListItems for the purpose of having another button component in an item, I saw some examples that I took as inspiration from the react docs https://mui.com/material-ui/react-list/  as for the "Unmerging" message, I never identified that as a UI bug, but its possible the styling tweaks I added brought it back to a normal state.